backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -4131,4 +4131,39 @@ databaseChangeLog:
                   descending: true
       rollback:
         - dropIndex:
-            indexName: ix__patient_link__created_at
\ No newline at end of file
+            indexName: ix__patient_link__created_at
+  - changeSet:
+      id: create-shedlock-table
```

Comment:
I know this PR is still in draft, but just a note that it's best practice for database changes to go into a separate PR - it makes rollbacks a bit easier/safer than bundling the changes together.

Reply:
created #4989 for this - thanks for the call out!